001-loginorangehrm/001a-loginwithoutframework/LogInPageOrangeHRM.java

```diff
@@ -0,0 +1,56 @@
+package projectorangehrm;
+
+import java.time.Duration;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class LogInPageOrangeHRM {
+
+	public static void main(String[] args) {
+		
+		//1.Launch Chrome Browser
+		WebDriver driver=new ChromeDriver();
+		
+		//maximize the browser(Method chaining)
+		driver.manage().window().maximize();
+		
+		//2. Navigate to OrangeHRM url
+		driver.get("https://opensource-demo.orangehrmlive.com/web/index.php/auth/login");

```

Comment:
dont hardcode it.. move this into properties file !   the way is : create a prop file and import that into this java file